recipes/llama-cpp-python/meta.yaml

```diff
@@ -35,9 +36,9 @@ requirements:
     - python
     - scikit-build-core >=0.5.1
     - pip
-    - cudatoolkit {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+    - cudatoolkit {{ cuda_compiler_version }}.*  # [cuda_compiler_version != "None"]
```

Comment:
Please don't depend on cudatoolkit but use the compilers.

Reply:
@xhochy, the compiler pinning is is not working for llama.cpp. The runtime requirements is going to 11.8 instead of holding at 11.2 which is what was used during the build.